### ISSUE TITLE
🐛 `ValueError: Number of time points in *.1D (0) is inconsistent with length of functional file *.nii.gz (182)`

### ISSUE THREAD

**Issue by shnizzedy** [FCP-INDI/C-PAC#1705]
### Describe the bug

> Seems to be hitting https://github.com/FCP-INDI/C-PAC/issues/996 or something like it more often with this updated config, for both of these regressors:
> https://github.com/FCP-INDI/C-PAC/blob/15c9d370edb59e1d54a4548a798a0a61b1425906/CPAC/resources/configs/pipeline_config_rbc-options.yml#L87-L144

\+ https://github.com/FCP-INDI/C-PAC/blob/15c9d370edb59e1d54a4548a798a0a61b1425906/CPAC/resources/configs/pipeline_config_rbc-options.yml#L146-L158

> [crash-20220405-180135-jclucas-build_nuisance_regressors.a0-18359279-7d06-4922-aa06-4a301d5ddb0d.txt](https://github.com/FCP-INDI/C-PAC/files/8421559/crash-20220405-180135-jclucas-build_nuisance_regressors.a0-18359279-7d06-4922-aa06-4a301d5ddb0d.txt)
> [crash-20220405-180139-jclucas-build_nuisance_regressors.a0-db15627d-d62d-47df-95da-283fb6eac933.txt](https://github.com/FCP-INDI/C-PAC/files/8421561/crash-20220405-180139-jclucas-build_nuisance_regressors.a0-db15627d-d62d-47df-95da-283fb6eac933.txt)
> [crash-20220405-191434-jclucas-build_nuisance_regressors.a0-55761957-2902-4e01-bcf9-b8e4d7cccc88.txt](https://github.com/FCP-INDI/C-PAC/files/8421562/crash-20220405-191434-jclucas-build_nuisance_regressors.a0-55761957-2902-4e01-bcf9-b8e4d7cccc88.txt)
> [crash-20220405-191435-jclucas-build_nuisance_regressors.a0-3f2f9170-f0e8-4380-a979-8a2c3b15e397.txt](https://github.com/FCP-INDI/C-PAC/files/8421563/crash-20220405-191435-jclucas-build_nuisance_regressors.a0-3f2f9170-f0e8-4380-a979-8a2c3b15e397.txt)
> [crash-20220405-191443-jclucas-build_nuisance_regressors.a1-e58e2fdd-0455-4947-be95-9c784fed8ba3.txt](https://github.com/FCP-INDI/C-PAC/files/8421564/crash-20220405-191443-jclucas-build_nuisance_regressors.a1-e58e2fdd-0455-4947-be95-9c784fed8ba3.txt)
> [crash-20220405-191443-jclucas-build_nuisance_regressors.a1-ffaf7100-cd86-4d6c-864f-71e8c1542ef9.txt](https://github.com/FCP-INDI/C-PAC/files/8421565/crash-20220405-191443-jclucas-build_nuisance_regressors.a1-ffaf7100-cd86-4d6c-864f-71e8c1542ef9.txt)

_Originally posted by @shnizzedy in https://github.com/FCP-INDI/C-PAC/issues/1683#issuecomment-1089287568_

### To reproduce

Run with https://github.com/FCP-INDI/C-PAC/blob/15c9d37/CPAC/resources/configs/pipeline_config_rbc-options.yml

### Expected behavior

The CSF 1D file is not all zeros

### Acceptance criteria

- [ ] The CSF 1D file is not all zeros
   and/or
- [ ] A warning or error is raised at the point of creating the completely-eroded segmentation mask

### C-PAC version

v1.8.4-dev

### Container platform

Docker, Singularity

### Additional context

I _think_ the issue arises from this section of the config: https://github.com/FCP-INDI/C-PAC/blob/15c9d370edb59e1d54a4548a798a0a61b1425906/CPAC/resources/configs/pipeline_config_rbc-options.yml#L146-L158

**Comment by shnizzedy** [comment]
I'm curious if we want `0.9` instead of `0.1` for `csf_erosion_prop`.

Docstrings:
https://github.com/FCP-INDI/C-PAC/blob/3d6f30e8705230081c65c6acec1479441633c88d/CPAC/resources/configs/pipeline_config_rbc-options.yml#L150-L154
https://github.com/FCP-INDI/C-PAC/blob/80424468c7f4e59c102f446b05d4154ec1cd4b2d/dev/docker_data/default_pipeline.yml#L1396-L1400
https://github.com/FCP-INDI/C-PAC/blob/80424468c7f4e59c102f446b05d4154ec1cd4b2d/CPAC/seg_preproc/utils.py#L198-L199
[**niworkflows/niworkflows/interfaces/probmaps.py**<br/>Lines 56 to 61 in 44408e60](https://github.com/nipreps/niworkflows/blob/44408e6062f39e3010cc74caa85b498f672ed1a4/niworkflows/interfaces/probmaps.py#L56-L61)
```Python
    mask_erode_prop = traits.Float(
        xor=["mask_erode_mm"], desc="erode input mask (target volume ratio)"
    )
    erode_prop = traits.Float(
        xor=["erode_mm"], desc="erode output mask (target volume ratio)"
    )
```

And the code looks to me like it erodes down _to_ `mask_erosion_prop`: https://github.com/FCP-INDI/C-PAC/blob/3d6f30e8705230081c65c6acec1479441633c88d/CPAC/seg_preproc/utils.py#L227-L232

**Comment by shnizzedy** [comment]
Resolved in #1683
